src/amo/components/SearchContextCard/index.js

```diff
@@ -25,10 +25,18 @@ export class SearchContextCardBase extends React.Component {
   render() {
     const { count, filters, i18n, loading } = this.props;
     const { query } = filters;
+    const { addonType } = filters;
 
     let searchText;
 
-    if (!loading && query) {
+    if (!loading && query && addonType) {
+      const addon = addonType === 'persona' ? 'theme' : addonType;
```

Comment:
Unfortunately `addonType` is not localised, so this won't work in any language that isn't English (and even then, `addonType` isn't very readible if it's something like `langpack`).

So this will need to be more of an `if`/`else` case like we have here: https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/SearchForm/index.js#L207

Sorry about that!

Reply:
Alright! I don't know why I didn't think of it. Sorry, my bad!

Reply:
No worries, we've made the same mistake too! 😉